homework/list.js

```diff
@@ -1,44 +1,37 @@
 'use strict';
 
-(function() {
+(function(exports) {
 
+ var list=function(){
   var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  var _wrapper = null;
+ }
+ 
+ list.prototype = {
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+  onNoteOpen(event) {
     if (event.target.classList.contains('note-title')) {
```

Comment:
And in fact it's a convention to prefix `on` to a handler with the corresponding event name. So this method should be renamed as `onNoteClicked`.